cypress/support/e2e.ts

```diff
@@ -18,3 +18,10 @@ before(function () {
 // HUB Component Port: 4202
 // EDA E2E Port: 4103
 // EDA Component Port: 4203
+
+Cypress.on('uncaught:exception', (_err, _runnable) => {
```

Comment:
Will this prevent all uncaught excepts from causing test failures?

Reply:
This will not catch excepts from tests.

Reply:
Needed this for the cypress **component** tests to work.
I was unable to get the cypress component test webpack config to build the webworkers.
This ignores the error that it cannot load them, and uses the inline support for handling the json and yaml that monaco has.